Examples/Swift/OrnamentsLayoutExample.swift

```diff
@@ -0,0 +1,59 @@
+import Mapbox
+
+// Ornament positions matrix array, this example will demonstrate how ornament looks like in different positions.
+let ornamentPosition: [[MGLOrnamentPosition]] = [
+    [.topLeft, .topRight, .bottomRight, .bottomLeft],
+    [.topRight, .bottomRight, .bottomLeft, .topLeft],
+    [.bottomRight, .bottomLeft, .topLeft, .topRight],
+    [.bottomLeft, .topLeft, .topRight, .bottomRight]
+]
+
+@objc(OrnamentsLayoutExample_Swift)
+class OrnamentsLayoutExample_Swift: UIViewController, MGLMapViewDelegate {
+    var mapView: MGLMapView!
+    var timer: Timer?
+    var currentPositionIndex: Int = 0
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.showsScale = true
+        mapView.setCenter(CLLocationCoordinate2DMake(39.915143, 116.404053), zoomLevel: 16, direction: 30, animated: false)
+        self.view.addSubview(mapView)
```

Comment:
Nit: omit `self` everywhere in this Swift implementation (unless forced to use it by being inside a block, etc.).